src/bugreporter/odemis_bugreporter.py

```diff
@@ -0,0 +1,520 @@
+        name_lbl = wx.StaticText(panel, wx.ID_ANY, "Name:")
+        name_ctrl = wx.TextCtrl(panel, wx.ID_ANY, size=(500, 23))
+        name_ctrl.Bind(wx.EVT_TEXT, self.on_name_text)
+        name_ctrl.Bind(wx.EVT_KEY_DOWN, self.on_name_key_down)
+        name_ctrl.Bind(wx.EVT_KILL_FOCUS, self.on_name_focus)
+        name_sizer.Add(name_lbl, 5, wx.EXPAND | wx.ALIGN_LEFT | wx.ALL, 10)
+        name_sizer.Add(name_ctrl, 10, wx.EXPAND | wx.ALIGN_LEFT | wx.ALL, 5)
+        sizer.Add(name_sizer)
+        email_sizer = wx.BoxSizer(wx.HORIZONTAL)
+        email_lbl = wx.StaticText(panel, wx.ID_ANY, "Email:")
+        email_ctrl = wx.TextCtrl(panel, wx.ID_ANY, size=(500, 23))
+        email_sizer.Add(email_lbl, 5, wx.EXPAND | wx.ALIGN_LEFT | wx.ALL, 10)
+        email_sizer.Add(email_ctrl, 10, wx.EXPAND | wx.ALIGN_LEFT | wx.ALL, 5)
+        sizer.Add(email_sizer)
+        summary_sizer = wx.BoxSizer(wx.HORIZONTAL)
+        summary_lbl = wx.StaticText(panel, wx.ID_ANY, "Summary:")
+        summary_ctrl = wx.TextCtrl(panel, wx.ID_ANY, size=(500, 23))
+        summary_sizer.Add(summary_lbl, 5, wx.EXPAND | wx.ALIGN_LEFT | wx.ALL, 10)
+        summary_sizer.Add(summary_ctrl, 10, wx.EXPAND | wx.ALIGN_LEFT | wx.ALL, 5)
+        sizer.Add(summary_sizer)
+        description_sizer1 = wx.BoxSizer(wx.HORIZONTAL)
+        description_sizer2 = wx.BoxSizer(wx.HORIZONTAL)
+        description_lbl = wx.StaticText(panel, wx.ID_ANY, "Description:")
+        default_txt = ("Ways to reproduce the problem:\n1.\n2.\n3.\n\nCurrent behaviour:\n\n" +
+                       "Expected behaviour:\n\nAdditional Information (e.g. reproducibility, severity):\n")
+        description_ctrl = wx.TextCtrl(panel, wx.ID_ANY, value=default_txt, size=(self.GetSize()[0], 400),
+                                       style=wx.TE_MULTILINE)
+        description_sizer1.Add(description_lbl, 5, wx.EXPAND | wx.ALIGN_LEFT | wx.ALL, 10)
+        description_sizer2.Add(description_ctrl, 5, wx.EXPAND | wx.ALIGN_LEFT | wx.LEFT | wx.RIGHT, 10)
+        sizer.Add(description_sizer1)
+        sizer.Add(description_sizer2)
+
+        # GDPR text
+        gdpr_sizer = wx.BoxSizer(wx.HORIZONTAL)
+        gdpr_lbl = wx.StaticText(panel, -1, GDPR_TEXT)
+        gdpr_lbl.Wrap(gdpr_lbl.GetSize().width)
+        font = wx.Font(10, wx.NORMAL, wx.ITALIC, wx.NORMAL)
+        gdpr_lbl.SetFont(font)
+        gdpr_sizer.Add(gdpr_lbl, 10, wx.EXPAND | wx.ALIGN_LEFT | wx.ALL, 10)
+        sizer.Add(gdpr_sizer, wx.EXPAND)
+
+        # Cancel and send report buttons
+        button_sizer = wx.BoxSizer(wx.HORIZONTAL)
+        cancel_btn = wx.Button(panel, wx.ID_ANY, "Cancel")
+        cancel_btn.Bind(wx.EVT_BUTTON, self.on_close)
+        button_sizer.Add(cancel_btn, 0, wx.ALL, 10)
+        report_btn = wx.Button(panel, wx.ID_ANY, "Report")
+        report_btn.Bind(wx.EVT_BUTTON, self.on_report_btn)
+        button_sizer.Add(report_btn, 0, wx.ALL, 10)
+
+        # Status update label (instead of throbber)
```

Comment:
Can you add a `TODO: replace by an animated throbber`.

BTW, why didn't you directly do it? It should be possible to embedded a GIF inside the python file. You can probably find a free version in Firefox's code.

Reply:
I tried to do it for some time, but I realized it would take me too long. I'm very busy with the robotics project at the moment, so maybe we can leave the todo for now. If you really want me to implement it, let me know.